app.js

```diff
@@ -1,17 +1,29 @@
 const express = require('express');
 const app = express();
-const { getApi, getTopics, getArticleById } = require('./controllers/api.controllers');
+const { getApi, getTopics, getArticles, getArticleById } = require('./controllers/api.controllers');
 
 app.get('/api', getApi);
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles', getArticles);
+
 app.get('/api/articles/:article_id', getArticleById);
 
+app.use((request, response, next) => {
+	const error = new Error('Not Found');
+	error.status = 404;
+	next(error);
+});
```

Comment:
This one is for dealing with invalid endpoint, might be better to use app.all() to handle it. Please take a look at the last example in NCNotes error-handling